code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -84,6 +84,16 @@
 	if(!beehome)
 		. += "<span class='warning'>This bee is homeless!</span>"
 
+/mob/living/simple_animal/hostile/poison/bees/ListTargets() // Bee processing is expessive, so we override them finding targets here.
+	if(!search_objects) //In case we want to have purely hostile bees
+		return ..()
+	else
+		. = list() // The following code is only very slightly slower than just returning oview(vision_range, targets_from), but it saves us much more work down the line
+		var/list/searched_for = oview(vision_range, targets_from)
+		for(var/obj/A in searched_for)
+			. += A
+		for(var/mob/A in searched_for)
+			. += A
 
```

Comment:
Undocumented changes?

Reply:
If you look at the original PR, this is a revert of an old /tg/ PR that changed Bee AI and broke the ListTargets proc behaviour.